astropy/time/core.py

```diff
@@ -503,8 +504,8 @@ def _get_time_fmt(self, val, val2, format, scale,
         guess available formats and stop when one matches.
         """
 
-        if (format is None and
-                (val.dtype.kind in ('S', 'U', 'O', 'M') or val.dtype.names)):
+        if (format is None
+                and (val.dtype.kind in ('S', 'U', 'O', 'M') or val.dtype.names)):
```

Comment:
FWIW, I think we should ignore W503. I don't see the value in that.

Reply:
PEP8 and Donald Knuth recommend breaking before binary operator.  https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator

I don't feel this needs to be a requirement, and indeed for years my linter made do it the other way (break *after*).  But I've come around after reading PEP8 and so will be adopting this style for my own development and would like to recommend this for table, time, io.ascii unless there is a reason otherwise.  It certainly doesn't need to be an astropy global standard.

Reply:
My vote is ignore for now but I can be overruled by other devs. 😸

Reply:
Like @taldcroft I used to do it the other way around, but I just did an autopep8 swap to break before binary operator in the two packages I have myself... I like it quite a bit better.

Here, though, I would also break before `or`

Reply:
I prefer to have it as an astropy global standard, but agree that we can skip enforcing it for now.